docs/manual/src/udl/ext_types_external.md

```diff
@@ -13,12 +13,12 @@ dictionary DemoDict {
 };
 ```
 
-And further, assume that you have another crate called `consuming-crate` which would like to use
-this dictionary. Inside `consuming-crate`'s UDL file you can reference `DemoDict` by using a
+And further, assume that you have another crate called `consuming_crate` which would like to use
```

Comment:
huh - I actually liked the `-` over the `_`, because Rust does magic with them. ie, if the crate *really* was `consuming-crate`, the distinction between `-` and `_` below becomes important.